Features/Transfer/Sources/ViewModels/ConfirmTransferSceneViewModel.swift

```diff
@@ -202,6 +202,71 @@ public final class ConfirmTransferSceneViewModel {
             selectedQuote: swapData.quote
         )
     }
+    
+    @SectionBuilder<TransferSection>
+    var listSections: [TransferSection] {
```

Comment:
Please consider creating a factory for building sections, as it can improve readability and reusability